presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/ResourceGroupManager.java

```diff
@@ -39,6 +39,8 @@
 
     ResourceGroupInfo getResourceGroupInfo(ResourceGroupId id);
 
+    List<ResourceGroupInfo> getAllResourceGroupInfo();
```

Comment:
If I don't see the return type, the method name could imply that you will return *all* the info about a single resource group. Maybe something like `getAllResourceGroups` or `getResourceGroupInfos` would be more clear?